### ISSUE TITLE
EllenTV extractor not working for some pages

### ISSUE THREAD

**Issue by TRox1972** [rg3/youtube-dl#10067]
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', 'http://www.ellentv.com/videos/1-szkgu2m2/']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.07.11
[debug] Git HEAD: 7f29cf5
[debug] Python version 3.5.1 - Darwin-15.3.0-x86_64-i386-64bit
[debug] exe versions: avconv 11.4, avprobe 11.4, ffmpeg 3.0.2, ffprobe 3.0.2, rtmpdump 2.4
[debug] Proxy map: {}
[EllenTV] 1-szkgu2m2: Downloading webpage
ERROR: Unable to extract partner id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/Users/Trox/dev/youtube-dl/youtube_dl/YoutubeDL.py", line 681, in extract_info
    ie_result = ie.extract(url)
  File "/Users/Trox/dev/youtube-dl/youtube_dl/extractor/common.py", line 347, in extract
    return self._real_extract(url)
  File "/Users/Trox/dev/youtube-dl/youtube_dl/extractor/ellentv.py", line 36, in _real_extract
    r"var\s+partnerId\s*=\s*'([^']+)", webpage, 'partner id')
  File "/Users/Trox/dev/youtube-dl/youtube_dl/extractor/common.py", line 650, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract partner id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by dstftw** [comment]
This video is not available.

**Comment by TRox1972** [comment]
@dstftw Yes it is.

```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-vF', u'kaltura:594251:1_szkgu2m2']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.07.11
[debug] Python version 2.7.11 - Darwin-15.3.0-x86_64-i386-64bit
[debug] exe versions: avconv 11.4, avprobe 11.4, ffmpeg 3.0.2, ffprobe 3.0.2, rtmpdump 2.4
[debug] Proxy map: {}
[Kaltura] 1_szkgu2m2: Downloading Kaltura signature
[Kaltura] 1_szkgu2m2: Downloading video info JSON
[Kaltura] 1_szkgu2m2: Downloading m3u8 information
[Kaltura] 1_szkgu2m2: Checking flv-400 video format URL
[Kaltura] 1_szkgu2m2: Checking mp4-312 video format URL
[Kaltura] 1_szkgu2m2: Checking mp4-305 video format URL
[Kaltura] 1_szkgu2m2: Checking 3gp-398 video format URL
[Kaltura] 1_szkgu2m2: Checking flv-580 video format URL
[Kaltura] 1_szkgu2m2: Checking hls-meta video format URL
[Kaltura] 1_szkgu2m2: Checking hls-312 video format URL
[Kaltura] 1_szkgu2m2: Checking hls-319 video format URL
[info] Available formats for 1_szkgu2m2:
format code  extension  resolution note
hls-meta     mp4        multiple   Quality selection URL
mp4-305      mp4        416x224     305k , isom container, avc1, 25fps, ~16.90MiB
hls-312      mp4        416x224     312k
mp4-312      mp4        416x224     312k , isom container, avc1, 25fps, ~21.60MiB
hls-319      mp4        416x224     319k
3gp-398      3gp        416x224     398k , 3gp4 container, mpeg-4 visual, 29fps, ~21.20MiB
flv-400      flv        426x239     400k , flash video container, vp6, 29fps, ~26.10MiB
flv-580      flv        416x224     580k , flash video container, vp6, 30fps, ~26.40MiB (best)
```

**Comment by dstftw** [comment]
Fixed in a0560d8ab83a565d502e160ccd993317a0ed69ef.
